smarts/core/chassis.py

```diff
@@ -59,9 +59,13 @@
 def _query_bullet_contact_points(bullet_client, bullet_id, link_index):
     contact_objects = set()
 
-    min_, max_ = bullet_client.getAABB(bullet_id, link_index)
     # `getContactPoints` does not pick up collisions well so we cast a fast box check on the physics
+    min_, max_ = bullet_client.getAABB(bullet_id, link_index)
+    # note that getAABB returns a box around the link_index link only,
+    # which means it's offset from the ground (min_ has a positive z)
```

Comment:
I see, that should mean that the only way the chassis hits the ground is if there is some serious loss of control or hitting inclined ground.

Reply:
Perhaps this is a mistake of thinking that the wheels are part of the chassis. The wheels cause elevation of the actual chassis so it never touches the ground. I believe this current behavior might be correct.

Reply:
Yep, that's what I just did a variation of (on the `map_abstraction` branch, as discussed in chat).